test/test_torch.py

```diff
@@ -10355,6 +10355,31 @@ def test_histc(self, device):
         self.assertEqual(
             torch.tensor([1], dtype=torch.float, device=device),
             actual)
+        # tensors with inf; min, max not provided -- should throw a RuntimeError
+        self.assertEqual(
+            torch.histc(torch.tensor([float("inf")], dtype=torch.float, device=device)),
+            "range of [inf, inf] is not finite")
```

Comment:
I think since this is raising an error not returning a string value, this check isn't going to work. you probably want assertRaisesRegex.